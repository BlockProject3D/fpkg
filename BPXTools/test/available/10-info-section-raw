export NAME="Info (Section)"
export COMMAND="-f test.bpx info -d 0"
export DESC="Test the info command"
export STATUS=1