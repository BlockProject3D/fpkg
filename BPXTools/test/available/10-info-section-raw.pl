$Test = {
    Name => "Info (Section)",
    Command => "-f test.bpx info -d 0",
    Description => "Test the info command",
    Status => 1
};

sub TestBegin {

}

sub TestEnd {

}
