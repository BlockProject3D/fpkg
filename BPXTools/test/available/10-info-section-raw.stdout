====> BPX Main Header <====
Type: P
Version: 1
File size: 3362131
Number of sections: 2
====> End <====

